samza-test/src/main/java/org/apache/samza/test/framework/TestRunner.java

```diff
@@ -369,4 +387,20 @@ public void run(Duration timeout) {
             new EndOfStreamMessage(null)));
       });
   }
+
+  private void deleteStoreDirectories() {
+    Preconditions.checkNotNull(configs.get(JobConfig.JOB_LOGGED_STORE_BASE_DIR()),
```

Comment:
What is this doing? Does checkNotNull take a vararg param?

Reply:
sanity check, if some users end up removing it,  yes it takes a vararg param

Reply:
Are you sure? The matching signature I see in my IDE is:
`public static <T> T checkNotNull(T reference, @Nullable Object errorMessage)`?
That wouldn't do what I think you want to do - check both configs for existence.